tests/scss/others/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -26,5 +69,26 @@ $display-breakpoints: map-deep-merge(
   $display-breakpoints
 );
 
+//list test
+$size: 2 3 4;
+@each $size in $sizes {
+  .icon- #{$size} {
+    border: "#{$size + "px"}" solid red;
+  }
+}
+
+//map test
+$font-weights: (
+  "regular": 400,
+  "medium": 500,
+  "bold": 700,
+);
+@each $name, $boldness in $icons {
+  .text- #{ $name } {
```

Comment:
Bug?

Reply:
I guess no. checked in the playground. Over there also it is formatting in the same way [link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASAZtGBaA7nASwHMALGAZwB0okACagCkpACc4iBXAGwEMXm6AFgAMw6gBp6UZgFs4AEwIcZA2gFZRk6swBGELvNXUA7KICUAbhDiQEAA4wC0cslB8WEXAAU+CFyh4uXB4ATxcbHRYeMABrOBgAZTtogigiZBgWDjgbOBkdBXkFABkeNI4eIjgAMQgWGR4YRzTkEB4OGAhrEDIZLgB1EgJ4cmSwOAS-YYIAN2GQ1rBycJBU8jgWGC8oogbkdED1mwArcgAPACEo2PiEnjli1Lh9w5yQU7OE1KIuOABFDgQeAvLhHEDJFjrFitchLFZ2FipGD9AjyGAkZAADmENgREHW-SidlaCLgUJmzxsAEdAfBtvZ-G1yNgoHAFApumwaQQ2NtKnskAdQW91jICBksiLvr8AUDnoLXjYYDwdCi0RikAAmJVRAhcb4AYQgMgFIDJam6HHWABUVf4hWCZtkAJJQIqwBJgREOACCboSMBCvxB6wAvqGgA)

Reply:
Let's add/fix two cases here in:
```
.text-#{ $name }
```

and

```
.text #{ $name }
```

Reply:
sorry, didn't get your point. Can you please explain a bit.
`.text-`
`#{ $name }`
is formatting into 
`.text- #{ $name }`

do you think it a bug? because we can't just assume that if it ends with "-", we have to merge the next text without space. The class name can also be just "test-", right?

Reply:
No, it is not bug, just put in tests:
```
@each 
$size 
in 
$sizes {
.icon
#{$size} 
{
border
: 
"#{$size + "px"}" 
solid 
red;
}
}
```

and

```
@each 
$size 
in 
$sizes {
.icon-#{$size} 
{
border
: 
"#{$size + "px"}" 
solid 
red;
}
}
```

and

```
//map test
$font-weights
: 
("regular": 400
, 
"medium": 500, 
"bold": 
700);
@each 
$name, 
$boldness 
in 
$icons {
.text
#{
$name
} {
color: red;
font-weight: 
"#{$boldness}"
}
}
```

```
//map test
$font-weights
: 
("regular": 400
, 
"medium": 500, 
"bold": 
700);
@each 
$name, 
$boldness 
in 
$icons {
.text-#{
$name
} {
color: red;
font-weight: 
"#{$boldness}"
}
}
```

Instead

Reply:
Thanks, will make the changes